### ISSUE TITLE
auth_token

### ISSUE THREAD

**Issue by vinnitu** [OnkelTem/ts-zello#2]
What about auth_token?

**Comment by OnkelTem** [comment]
Как его получить вот тут написано: https://github.com/zelloptt/zello-channel-api/blob/master/AUTH.md

P.S. Если что, я долгое время не смотрел как дела у этой либы. Причина - никому это оказалось не надо, а я, удовлетворив свой интерес, перестал библиотекой заниматься.

Но если будет интерес, можно реанимировать :)

**Comment by vinnitu** [comment]
Не пытались разобраться как приложение получает информацию о пользователях? О изменение их статуса в канале? В процессе регистрации? Я понимаю, что через ws не сделаешь, но приложение куда-то и как-то делает запросы...

**Comment by OnkelTem** [comment]
Нет, такой задачи не стояло. По сокетам даже список пользователей не приходит, только если чел что-то прислал и начал говорить. Административных событий в API тоже нет. Zello обещали расширить функционал своего API, но кажется так и забили.

**Comment by vinnitu** [comment]
да, я понимаю, но ведь приложение может - значит есть способ
и административные события тоже интересны
